mkt/commonplace/views.py

```diff
@@ -116,12 +116,12 @@ def commonplace(request, repo, **kwargs):
     # need it.
     fxa_auth_state, fxa_auth_url = fxa_auth_info()
     site_settings = {
+        'dev_pay_providers': settings.DEV_PAY_PROVIDERS,
```

Comment:
anything that runs on Firefox OS won't need this because all that is handled by device settings. It would be nice if we could change device settings but you have to do that at build time (settings similar to these are baked into engineering builds).